dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScope.kt

```diff
@@ -77,8 +75,18 @@ internal class RumSessionScope(
 
             onApplicationDisplayed(event, viewScope, actualWriter)
             activeChildrenScopes.add(viewScope)
-        } else if (activeChildrenCount == 0) {
-            devLogger.w(MESSAGE_MISSING_VIEW)
+        } else if (activeChildrenScopes.size == 0 && (
+                event is RumRawEvent.AddError ||
+                    event is RumRawEvent.AddLongTask ||
+                    event is RumRawEvent.StartAction ||
```

Comment:
Not sure if actions make sense for background views. What would that mean if a "tap" action is registered for a background view?

Reply:
Think about a music application with a foreground service, at least this is common in Android. You will have `TAP` on the Play/Stop/Pause..etc.